easyWslLib/DockerDownloader.cs

```diff
@@ -54,7 +62,7 @@ public async Task DownloadImage(string distroImage, Action<HttpProgress> httpPro
 
             string repository = "";
             string tag = "";
-            string registry = "LLL";
+            string registry = "registry-1.docker.io";
```

Comment:
I wasn't sure of the purpose of LLL, I could not get it to work with that value.